Makefile

```diff
@@ -1,66 +1,66 @@
 #
```

Comment:
Also, I think that the packaging script here should also be a target from the Makefile so you can just run `make package` and it builds a release archive of the binaries (so basically just run the script you have and then zip up the directory)